client/app/components/manager-navbar/manager-navbar.module.js

```diff
@@ -85,7 +69,74 @@ angular.module("App").run((LANGUAGES, managerNavbar, User, constants, translator
                         }
                     }
                 ]
+            });
+
+            return generalMenuItems;
+        }
+
+        function buildAssistanceMenuItem (currentUser) {
+            const currentSubsidiaryURLs = constants.urls[currentUser.ovhSubsidiary];
+
+            const assistanceMenuItems = [];
+            if (_(currentSubsidiaryURLs).has("guides.home")) {
+                assistanceMenuItems.push({
+                    label: translator.tr("common_menu_support_all_guides"),
+                    url: currentSubsidiaryURLs.guides.home
+                });
             }
-        ]);
+
+            assistanceMenuItems.push({
+                label: translator.tr("common_menu_support_new_ticket"),
+                click: () => {
+                    if (!OtrsPopupService.isLoaded()) {
+                        OtrsPopupService.init();
+                    } else {
+                        OtrsPopupService.toggle();
+                    }
+                }
+            });
+
+            assistanceMenuItems.push({
+                label: translator.tr("common_menu_support_list_ticket"),
+                url: "#/support"
+            });
+
+            if (_(currentSubsidiaryURLs).has("support_contact")) {
+                assistanceMenuItems.push({
+                    label: translator.tr("common_menu_support_telephony_contact"),
+                    url: currentSubsidiaryURLs.support_contact
+                });
+            }
+
+            return {
+                label: translator.tr("common_menu_support_assistance"),
+                subLinks: assistanceMenuItems
+            };
+        }
+
+        function getCurrentLanguageName () {
+            const currentLanguage = _(LANGUAGES).find({ value: translator.getLanguage() });
+            const currentLanguageName = _(currentLanguage).get("name");
+
+            return currentLanguageName;
+        }
+
+        function getLanguageChangingMenuObject () {
+            const languageChangingMenuObject = _(LANGUAGES)
+                .filter((language) => _(language).has("name", "value"))
+                .map((language) => ({
+                    label: language.name,
+                    click: () => {
+                        localStorage["univers-selected-language"] = language.value;
+                        window.location.reload();
+                    },
+                    lang: _(language.value).chain()
```

Comment:
`_(language.value).chain()` === `_(language.value)`

Reply:
You should read the remaining of the code:
_(language.value).chain()
    .words()
    .head()
    .value()

If I didn't use `.chain()` I would have had to do:

`_.head(_.words(language.value))`

Reply:
3 vs 4 my bad :(